v-next/hardhat/src/internal/cli/init/subprocess.ts

```diff
@@ -0,0 +1,24 @@
+import type { LastParameter } from "../../../types/utils.js";
+
+import { spawn as nodeSpawn } from "node:child_process";
+
+export async function spawn(
+  command: string,
+  args: string[],
+  options: LastParameter<typeof nodeSpawn>,
+): Promise<void> {
+  console.log(command, args.join(" "));
```

Comment:
You forgot this debug log

Reply:
Logging the command that is to be executed was intentional because I felt it communicated better to the user what's happening, but I do 100% agree that it looks odd here. I moved the `console.log` to the caller now where we have all the other UI handling. It definitely feels much more in-place there.